README.rst

```diff
@@ -1,4 +1,4 @@
-Deep Learning Tutorials
+please include in titles Machine learning and Deep Learning Tutorials
```

Comment:
Why change the title? This site isn't updated, so I do not think any of us want to change the title.